libsolidity/analysis/ReferencesResolver.cpp

```diff
@@ -283,4 +299,58 @@ void ReferencesResolver::operator()(yul::VariableDeclaration const& _varDecl)
 		visit(*_varDecl.value);
 }
 
+void ReferencesResolver::resolveInheritDoc(StructuredDocumentation const& _documentation, StructurallyDocumentedAnnotation& _annotation)
+{
+	switch (_annotation.docTags.count("inheritdoc"))
+	{
+	default:
+		m_errorReporter.docstringParsingError(
+			5142_error,
+			_documentation.location(),
+			"Documentation tag @inheritdoc can only reference one contract."
```

Comment:
Strictly, all tags could refer to the same contract. So maybe: ... can only be given once?